src/Components/NewProjectPage/index.js

```diff
@@ -166,24 +167,76 @@ export default class NewProjectPage extends React.Component {
   }
 
   renderSaveSuccess() {
-    if (this.state.status==="saved") {
+    if (this.state.status === "saved") {
       return <div data-test="project-update-success">Project updated!</div>;
     }
   }
-  renderSubmitSuccess() {
-    if(this.props.status==="LA Draft") {
-      return <div data-test="project-create-success">Project created!</div>;
+
+  getProjectLink() {
+    return <a href={this.getProjectURL()}>{this.getProjectURL()}</a>;
+  }
+
+  getProjectURL() {
+    let path = window.location.href;
+    let endChar = path.includes("?") ? path.indexOf("?") : path.length;
+    return path.substr(0, endChar);
+  }
+
+  getEmailSubject() {
+    let type = this.props.projectType ? this.props.projectType : "";
+    return `Your ${type.toUpperCase()} Project`;
+  }
+
+  getEmailBody() {
+    return `Follow this link to view your project: ${this.getProjectURL()};`;
```

Comment:
```suggestion
    return `Follow this link to view your project: ${this.getProjectURL()}`;
```
You need to remove the ; from inside the string as it's being posted into the email body.